installer/go/README.md

```diff
@@ -0,0 +1,45 @@
+# FlowFuse Device Agent Installer
+
+This directory contains the Go-based installer for the FlowFuse Device Agent.
+
+## Building
+
+To build the installer locally:
+
+```bash
+# Navigate to the installer directory
+cd installer/go
+
+# Execute build script
+./build.sh
+```
```

Comment:
This is now outdated. 

Needs to list the make commands

Also, needs to mention any dependancies like `make` and `sed` and how to install them 

For windows, this is:

```
winget install --id=GnuWin32.Make  -e
winget install --id=mbuilov.sed  -e
```
And after install, the user (or system path) will need to include `C:\Program Files (x86)\GnuWin32\bin`